source/uwp/Renderer/lib/AdaptiveToggleInput.h

```diff
@@ -25,6 +25,9 @@ namespace AdaptiveNamespace
         IFACEMETHODIMP get_Title(_Outptr_ HSTRING* title);
         IFACEMETHODIMP put_Title(_In_ HSTRING title);
 
+        IFACEMETHODIMP get_Wrap(_Outptr_ boolean* wrap);
```

Comment:
>_Outptr_ [](start = 32, length = 8)

Should be _Out_

Reply:
what's reason for using Out insteand of Outptr?

---
In reply to: [245450087](https://github.com/Microsoft/AdaptiveCards/pull/2264#discussion_r245450087) [](ancestors = 245450087)

Reply:
Basically outptr is for when the out parameter is a pointer. From the sal header:

"_Out_  : output parameter, written to by called function, pointed-to location not expected to be initialized prior to call
_Outptr_ : like _Out_ when returned variable is a pointer type(so param is pointer-to-pointer type). Called function provides/allocated space."

---
In reply to: [245450521](https://github.com/Microsoft/AdaptiveCards/pull/2264#discussion_r245450521) [](ancestors = 245450521,245450087)